index.js

```diff
@@ -207,6 +213,16 @@ function flushNonces() {
 	fs.writeFile("nonces.json", JSON.stringify(nonces), () => {});
 }
 
+function checkPassword(password) {
+	for (const validPassword of config.password) {
+		if (crypto.createHash("sha256").update(password).digest("hex") === validPassword) {
```

Comment:
Why not do the crypto stuff outside of loop instead? Results of crypto are same on each loop after all.

Reply:
Oh yeah whoops.